app/models/enum/analysis.py

```diff
@@ -0,0 +1,37 @@
+from aenum import Enum
```

Comment:
you want to use the regular `enum` package from the standard library here.
I used the `aenum` package for dynamic Enums only.